docs/concepts/crd-upgrade-safety.md

```diff
@@ -61,18 +61,18 @@ kind: ClusterExtension
 metadata:
     name: clusterextension-sample
 spec:
+    namespace: default
+    serviceAccount:
+      name: argocd-installer
     source:
       sourceType: Catalog
       catalog:
         packageName: argocd-operator
         version: 0.6.0
     install:
-      namespace: default
-      serviceAccount:
-        name: argocd-installer
       preflight:
         crdUpgradeSafety:
-          disabled: true
+          enforcement: "None"
```

Comment:
Do we actually want the quotation marks here for this string? or does it not matter?

Reply:
You changed the spec but not the test on top of it:

![Screenshot 2024-11-11 at 18 01 02](https://github.com/user-attachments/assets/5b1b490b-e4c8-45dc-9297-063e54cd9e7d)

^ c/c @trgeiger

Reply:
> Do we actually want the quotation marks here for this string? or does it not matter?

I'll check. It might have been a copy-paste error.


> You changed the spec but not the test on top of it:

Good catch! I'll update the string above as well.

Reply:
I just actually realised that I haven't gone through this page properly: I haven't tested it yet. I only gone through "Getting Started" and "Tutorials". I reverted this and we can address other sections later.

Resolving.